py/server/tests/test_parquet.py

```diff
@@ -384,7 +384,7 @@ def time_test_helper(pa_table, new_schema, dest):
             # Write the provided pyarrow table type-casted to the new schema
             pyarrow.parquet.write_table(pa_table.cast(new_schema), dest)
             from_disk = read(dest)
-            df_from_disk = to_pandas(from_disk)
+            df_from_disk = to_pandas(from_disk, dtype_backend=None)
             original_df = pa_table.to_pandas()
             # Compare the dataframes as strings
             self.assertTrue((df_from_disk.astype(str) == original_df.astype(str)).all().values.all())
```

Comment:
Isn't this `astype(str)` stuff indicating a problem in the test or library?  If the new default backend is used, shouldn't this be unneeded?

Reply:
See the response above.